### PR TITLE
test: Adds tests to the CustomFrame component

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Adds tests to the `CustomFrame` component.

### TEST PLAN
1 - Execute `CustomFrame` tests.
2 - All tests should pass.

@rusackas @junlincc 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/explore/components/controls/DateFilterControl/components/CustomFrame.test.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import { CustomFrame } from 'src/explore/components/controls/DateFilterControl/components/CustomFrame';
```
Comment (geido) [inline superset-frontend/src/explore/components/controls/DateFilterControl/components/CustomFrame.test.tsx:22]: I think it is safe to just use a relative path here?

**Review by geido — APPROVED** [review]
LGTM! with just a microscopic nit

File: superset-frontend/src/explore/components/controls/DateFilterControl/components/CustomFrame.test.tsx (hunk 1 above)
Comment (michael-s-molina) [reply superset-frontend/src/explore/components/controls/DateFilterControl/components/CustomFrame.test.tsx]: Fixed.

**Review by rusackas — APPROVED** [review]
Looks awesome, thank you!

**Pull request merged**